pages/meldInteresse.js

```diff
@@ -0,0 +1,143 @@
+  color: ${props => props.theme.secondary};
+  font-size: 16px;
+  resize: none;
+  border: 1px solid #c4c4c4;
+  box-sizing: border-box;
+  border-radius: 5px;
+  padding: 5px;
+  :invalid {
+    box-shadow: none;
+  }
+`;
+
+const TextareaDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const StyledSubmitButton = styled.input`
+  color: ${props => props.theme.secondary};
+  font-size: 25px;
+  border-radius: 15px;
+  width: 15%;
+  background-color: ${props => props.theme.primary};
+  border-color: ${props => props.theme.secondary};
+  padding: 15px;
+  align-self: flex-end;
+  transition-duration: 250ms;
+  transition-property: background-color;
+  :hover {
+    background-color: gray;
+  }
+`;
+
+const InfoLabel = styled.label`
+  font-size: 15px;
+`;
+
+const MeldInteresse = ({ className }) => {
+  const checkboxList = [
+    { text: 'ITEX', textOnHover: 'IT-eksursjonen' },
+    { text: 'Bedriftspresentasjon', textOnHover: 'bedpress' },
+    { text: 'Kurs / faglig arrengement', textOnHover: 'kurs' },
+    { text: 'Annonse i Offline', textOnHover: 'reklame' },
+    { text: 'Artikkel i Offline', textOnHover: 'artikkel' },
+    { text: 'Techtalks', textOnHover: 'Tekniske taler' },
+    { text: 'Lorem Ipsum', textOnHover: 'Ipsum Lorem' },
+    { text: 'Karrieremuligheter', textOnHover: 'Jobb' },
+    { text: 'foobar', textOnHover: 'foo' },
+  ];
+
+  return (
+    <>
+      <form className={className} action="POST" id="meldInteresseForm">
+        <HeaderStyling>Meld interesse:</HeaderStyling>
+        <FormWrapper>
+          <InfoLabel>Felt merket med en gul stjerne er nødvendige at du fyller ut</InfoLabel>
+          <InputField label="Bedrift" type="text" name="Bedrift" placeholder="Navn på bedriften..." />
+          <InputField label="Kontaktperson" type="text" name="Kontaktperson" placeholder="Navn på kontaktperson..." />
+          <InputField label="E-post" type="email" name="Epost" placeholder="E-posten det ønskes svar til..." />
+          <StyledLabel>Huk av det du er interessert i</StyledLabel>
+          <InfoLabel>Du kan holde musepekeren over de forskjellige feltene for mer informasjon</InfoLabel>
+          <CheckboxWrapper>
+            {checkboxList.map((el, i) => (
+              <Checkbox name={'Interests'} value={el.text} key={i} title={el.textOnHover} />
+            ))}
+          </CheckboxWrapper>
+          <TextareaDiv>
+            <StyledLabelHeader htmlFor="bedriftKommentar">Kommentarer</StyledLabelHeader>
+            <StyledTextarea
+              id="bedriftKommentar"
+              name="bedriftKommentar"
+              placeholder="Utdypning av ønsker og hvordan de tenkes gjennomført..."
+              required
+            />
+          </TextareaDiv>
+          <StyledSubmitButton type="submit" value="Send inn" />
+        </FormWrapper>
+      </form>
+      <Circles />
+    </>
+  );
+};
+
+const MeldInteresseStyle = styled(MeldInteresse)`
```

Comment:
We should maybe include stuff which is kinda universal, like 
```css
label {
    font-size: 36px;
} 
```
since a lot of labels use a common style. Also multiple other things which might be repeated in the css

Reply:
Could also create a global style-sheet imported at the root of the application with things like this. If it's something that's constantly being repeated, then perhaps it's better to default to this behaviour, and rather override it when necessary.

Reply:
True, 
```css
  ${Label} {
    font-size: 36px;
  }
```
is what i recently put in the meldinteresse form, but most likely this is gonna be reused so we might as well put this part in the GlobalStyle